server/routes/files.js

```diff
@@ -35,4 +36,53 @@ router.get('/media-files', (req, res) => {
   });
 });
 
+router.get('/stream', (req, res) => {
+  let filePath = req.query.path;
+  fs.stat(req.query.path, (err, stats) => {
+    if (err) {
+      if (err.code == 'ENOENT') {
+        return res.status(statusCodes.NOT_FOUND).json({message: 'File not found at path: ' + filePath});
+      }
+      return res.status(statusCodes.INTERNAL_SERVER_ERROR).json({message: err.message});
+    }
+    let range = req.headers.range;
+    if (!range) {
+      return res.status(statusCodes.REQUESTED_RANGE_NOT_SATISFIABLE).json({message: 'Range not found'});
+    }
+    let positions = range.replace(/bytes=/, '').split('-');
+
+    let start = parseInt(positions[0], 10);
+    let fileSize = stats.size;
+    let end = positions[1] ? parseInt(positions[1], 10) : fileSize - 1;
+
+    let chunkSize = (end - start) + 1;
+    let mimeType = fileExplorer.getMimeType(filePath);
+    if (!mimeType) {
+      return res.status(400).json({message: 'Unsupported video format'});
```

Comment:
`statusCodes. BAD_REQUEST`

Reply:
Is there a benefit using this library? Looking up its representations for simple status codes is inconvenient sometimes.